externs/lcevc.js

```diff
@@ -0,0 +1,93 @@
+/**
+* @fileoverview Externs for LcevcDil
+* compiler.
+*
+* @externs
+*/
+'use strict';
+
+var libDPIModule = {};
+var LcevcDil = {};
+
+/**
+* LCEVC DIL constructor
+* @constructor
+*/
+LcevcDil.LcevcDIL = class {
+  /**
+     *  @param {HTMLVideoElement} media
+     *  @param {HTMLCanvasElement} canvas
+     *  @param {Object=} dilConfig
+     */
+  constructor(media, canvas, dilConfig) {
+    /** @type {number} */
+    this.aspectRatio;
+
+    /** @type {Boolean} */
+    this.isFullscreen;
+
+    /** @type {Object} */
```

Comment:
This should also be more specific if it's for public use.  Is this perhaps an instance of `LcevcDil.SupportObject`?

Reply:
with the change in `Lcevc.Dil.SupportObject` this is not relevant anymore. so this has been removed.